lib/compat/wordpress-6.0/class-gutenberg-rest-global-styles-controller.php

```diff
@@ -231,7 +231,11 @@ public function get_theme_items( $request ) {
 			foreach ( $nested_html_files as $path => $file ) {
 				$decoded_file = wp_json_file_decode( $path, array( 'associative' => true ) );
 				if ( is_array( $decoded_file ) ) {
-					$variations[] = ( new WP_Theme_JSON_Gutenberg( $decoded_file ) )->get_raw_data();
+					$variation = ( new WP_Theme_JSON_Gutenberg( $decoded_file ) )->get_raw_data();
+					if ( ! isset( $variation['name'] ) ) {
```

Comment:
Not sure if it is handled anywhere else, but maybe want to make this an empty check instead? That way you can't pass:

```
"name": ""
```